### DISCUSSION TITLE
404 Errors

### DISCUSSION THREAD

**Discussion by getsean** [DiscipleTools/disciple-tools-theme#1109] [Q&A] [answered]
I have a new install, I saw on the documentation to go to permalinks and click save if you can't get to the /contacts page but I still can't get the theme to show anything on the front end. Any other thoughts? It is activated just won't load.  

http://51.222.141.102/contacts

Any help would be great thanks!

**Comment by corsacca** [comment, marked as answer]
Hey, @getsean . Sorry you are experiencing issues on setup.

I think you are referencing this page: https://disciple.tools/dev-docs/hosting/404-errors-on-new-install/
Note: Sometimes you need to hit "Save Changes" on the permalinks page twice.

If that does not work, it could be a .htaccess issue. 
I'd start by looking at that, base on this stackoverflow:
https://stackoverflow.com/questions/13149418/the-requested-url-about-was-not-found-on-this-server
Don't forget to restart apache (or the server).

**Reply by getsean** [reply]
Thanks so much for the quick reply. I will check out that article but I did ended up getting it working on a digital ocean server. 

https://marketplace.digitalocean.com/apps/wordpress

It worked great and starts at $5 a month. So will host it there for now. 

Again thanks!
